admin-api-server/test/app.e2e.spec.ts

```diff
@@ -2537,74 +2576,95 @@ describe('AppController (e2e)', () => {
           count: 7,
           data: [
             {
-              amount: 1,
               from: null,
               id: 1,
               kind: 'sale',
               price: '0.10',
               timestamp: 808223640,
               to: 'addr',
               tokenId: 1,
+              conversion_rate: '0.01',
+              currency: 'EUR',
+              edition_size: 1,
+              transaction_value: '0.00',
             },
             {
-              amount: 1,
               from: null,
               id: 2,
               kind: 'sale',
               price: '7.60',
               timestamp: 808223640,
               to: 'addr',
               tokenId: 30,
+              conversion_rate: '0.01',
+              currency: 'EUR',
+              edition_size: 1,
+              transaction_value: '0.04',
             },
             {
-              amount: 1,
               from: null,
               id: 3,
               kind: 'sale',
               price: '4.30',
               timestamp: 817171200,
               to: 'addr',
               tokenId: 4,
+              conversion_rate: '0.00',
+              currency: 'EUR',
+              edition_size: 1,
+              transaction_value: '0.01',
             },
             {
-              amount: 1,
               from: null,
               id: 4,
               kind: 'sale',
               price: '9.80',
               timestamp: 817171200,
               to: 'addr',
               tokenId: 7,
+              conversion_rate: '0.00',
+              currency: 'EUR',
+              edition_size: 1,
+              transaction_value: '0.02',
             },
             {
-              amount: 1,
               from: null,
               id: 5,
               kind: 'sale',
               price: '9.20',
               timestamp: 817171200,
               to: 'addr',
               tokenId: 10,
+              conversion_rate: '0.00',
+              currency: 'EUR',
+              edition_size: 1,
+              transaction_value: '0.02',
             },
             {
-              amount: 1,
               from: null,
               id: 6,
               kind: 'sale',
               price: '7.80',
               timestamp: 817802640,
               to: 'tz1',
               tokenId: 2,
+              conversion_rate: '0.00',
```

Comment:
you are right we need to change the data under testing here because this isn't showing very useful behavior

Reply:
Now they are not 0.00 anymore.